opportunities/index.html

```diff
@@ -35,8 +35,17 @@ <h3 >Researchers and Developers</h3>
         <header>
           <h3>Other opportunities</h3>
         </header>
-        <p>
-          We currently do not offer a course or fellow program. However, we will announce it here as soon as this changes. So check back! In the meantime, take a look at <a href="past_opportunities.html">past opportunities</a>.
```

Comment:
Let's just comment this line out and not remove it, so that we can put it back in, when the course post isn't relevant any longer.

Reply:
This line has been commented when I first made the pull request. I double checked and on the story/WEB-15 branch, it is commented out.

Reply:
this line needs to be commented out in here or it will be removed from the master branch.